extensions/cornerstone/src/Viewport/Overlays/ViewportOrientationMarkers.tsx

```diff
@@ -137,7 +137,7 @@ function ViewportOrientationMarkers({
       }
 
       const imageIndex = imageSliceData.imageIndex;
-      const imageId = viewportData?.imageIds[imageIndex];
+      const imageId = viewportData.imageIds?.[imageIndex];
```

Comment:
The viewportData is already null checked, but the imageIds isn't, so move the check onto imageIds.